pkg/cmd/marathon/marathon_about.go

```diff
@@ -11,7 +14,19 @@ func newCmdMarathonAbout(ctx api.Context) *cobra.Command {
 		Use:   "about",
 		Short: "Print info.json for DC/OS Marathon.",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return python.InvokePythonCLI(ctx)
+			client, err := marathon.NewClient(ctx)
```

Comment:
I'd prefer to have this function consist of a single function call just like in https://github.com/dcos/dcos-core-cli/blob/2bc9f1b3f61fe50a4c863ae2f176c30815d473b8/pkg/cmd/pkg/package_install.go#L38 and then let's test this functionality as well, please.